canvasapi/account.py

```diff
@@ -616,6 +616,69 @@ def list_enrollment_terms(self, **kwargs):
             **combine_kwargs(**kwargs)
         )
 
+    def list_user_logins(self, **kwargs):
+        """
+        Given a user ID, return that user's logins for the given account.
+
+        :calls: `GET /api/v1/accounts/:account_id/logins \
+        <https://canvas.instructure.com/doc/api/logins.html#method.pseudonyms.index>`_
+
+        :rtype: :class:`canvasapi.paginated_list.PaginatedList` of
+            :class:`canvasapi.account.Login`
```

Comment:
`canvasapi.account.Login` is incorrect. Since the Login class is in login.py, it should be `canvasapi.login.Login`